### PR TITLE
Fusion: S1: Add damage values for lava rooms

### PR DESCRIPTION
**Pull request by shark20061**
These are as of the current (60% reduction) implementation of the suit logic, which seems unlikely to change now.

### CODE REVIEW

File: randovania/games/fusion/logic_database/Sector 1 SRX.txt [hunk 1]
```diff
@@ -574,12 +574,17 @@ Extra - room_id: [17]
   > Door to Lava Lake Annex
       Any of the following:
           Gravity Suit and Varia Suit
-          Damage Runs (Intermediate) and Lava Damage ≥ 70
+          Damage Runs (Intermediate) and Lava Damage ≥ 60
+          Varia Suit and Damage Runs (Beginner) and Lava Damage ≥ 21
+          Gravity Suit and Damage Runs (Beginner) and Lava Damage ≥ 30
   > Door to Zebesian Tower
```
Comment (Miepee) [inline randovania/games/fusion/logic_database/Sector 1 SRX.txt:580]: What about jump extends?

File: randovania/games/fusion/logic_database/Sector 1 SRX.txt [hunk 2]
```diff
@@ -588,12 +593,22 @@ Extra - room_id: [17]
   * Extra - room: 17
   * Extra - blockx: 44
   * Extra - blocky: 19
+  > Door to Lava Lake Annex
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Gravity Suit and Varia Suit
```
Comment (Miepee) [inline randovania/games/fusion/logic_database/Sector 1 SRX.txt:600]: grav+varia is unnecesary, as you can already reach the door by going to upper door first and then free lava diving to the lower left

File: randovania/games/fusion/logic_database/Sector 1 SRX.txt (hunk 1 above)
Comment (shark20061) [reply randovania/games/fusion/logic_database/Sector 1 SRX.txt:580]: Getting regular wall jump physics means that jump extends don't help anymore - unspinning will negate the extra horizontal movement the wall jump grants.

File: randovania/games/fusion/logic_database/Sector 1 SRX.txt (hunk 2 above)
Comment (shark20061) [reply randovania/games/fusion/logic_database/Sector 1 SRX.txt:600]: That's true, will remove

File: randovania/games/fusion/logic_database/Sector 1 SRX.txt (hunk 1 above)
Comment (Miepee) [reply randovania/games/fusion/logic_database/Sector 1 SRX.txt:580]: There isnt any mention of wall jumps in there tho? Unless they also apply to normal jumps

Comment (shark20061) [reply randovania/games/fusion/logic_database/Sector 1 SRX.txt:580]: Oof, this terminology is confusing. I should have said "kick jump physics", which is any jump that involves kicking off of the wall. In addition to the "wall jump", there is also the kick jumps off of ladders and  from ledge grabs, which use the same physics as a wall jump. In this case, the jump used here is a kick jump by hanging off the cliffside ledge.

Comment (Miepee) [reply randovania/games/fusion/logic_database/Sector 1 SRX.txt:580]: > the jump used here is a kick jump by hanging off the cliffside ledge.

Then this should probably have beginner movement attached to it IMO.

Comment (shark20061) [reply randovania/games/fusion/logic_database/Sector 1 SRX.txt:580]: Sure, and added damage values for not getting the jump to the platform

**Review by JeffGainsNGames — APPROVED** [review]

**Pull request merged**